### PR TITLE
wsd: added config option for background manualsave

### PR DESCRIPTION
**Pull request by Rash419**
Change-Id: I76de35fe9b4135b74a96436f0c9a3245f4e00bd9

* Target version: master

### CODE REVIEW

File: wsd/DocumentBroker.cpp [hunk 1]
```diff
@@ -2743,7 +2744,8 @@ bool DocumentBroker::sendUnoSave(const std::shared_ptr<ClientSession>& session,
 
     // Note: It's odd to capture these here, but this function is used from ClientSession too.
     bool autosave = isAutosave || (_unitWsd && _unitWsd->isAutosave());
-    bool background = forceBackgroundSave || (autosave && _backgroundAutoSave);
```
Comment (mmeeks) [inline wsd/DocumentBroker.cpp:2746]: So - for the very last save of the document - we don't want to do a background save. Can you check from the logs that if you modify a document, then close the window without saving - that the final 'force' save is not a background save - even with the settings turned on.

Thanks!

**Review by mmeeks — CHANGES_REQUESTED** [review]
We also need to dig into the wopi save unit test to see what is going on there. Perhaps it is best to get this setting included - but disabled initially as/when you've cleaned up the comment =)

Thanks!

**Comment by mmeeks** [comment]
I'll work on finishing and merging this then.

**Comment by Rash419** [comment]
closing this in favor of https://github.com/CollaboraOnline/online/pull/9098

**Pull request closed without merging**